### PR TITLE
feat: expose enhanced monitoring and performance insights

### PR DESCRIPTION
**Pull request by zucchinidev**
[#183023673](https://www.pivotaltracker.com/story/show/183023673)

[#183023669](https://www.pivotaltracker.com/story/show/183023669)

### Checklist:

* [x] Have you added Release Notes in the docs repositories?
* [x] Have you followed the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)?

### CODE REVIEW

**Review by FelisiaM — APPROVED** [review]

File: aws-mysql.yml [hunk 1]
```diff
@@ -265,6 +265,35 @@ provision:
...
+    details: |
+      Enhanced Monitoring requires permission to act on your behalf to send OS metric information to CloudWatch Logs.
+      This property represents the ARN for the IAM role that permits RDS to send enhanced monitoring metrics to CloudWatch Logs.
+      To read about setting up and enabling Enhanced Monitoring see https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_Monitoring.OS.Enabling.html.
+    default: ""
+  - field_name: performance_insights_enabled
+    type: boolean
+    details: Specifies whether Performance Insights are enabled (see https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_PerfInsights.Overview.html).
+    default: false
+  - field_name: performance_insights_kms_key_id
```
Comment (FelisiaM) [inline aws-mysql.yml:290]: There is another performance insight property "performance_insights_retention_period" that I think would be useful to also expose. 
I think this was missed for postgres, but we have the opportunity to add it for both now.

Comment (FelisiaM) [reply aws-mysql.yml:290]: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#performance_insights_retention_period

Comment (FelisiaM) [reply aws-mysql.yml:290]: @zucchinidev Maybe that can be done as a separate PR and this can be merged as it all looks good.

Comment (zucchinidev) [reply aws-mysql.yml:290]: @FelisiaM, What you say sounds great. If it’s okay, I'll add it to this PR and also check the Terraform tests, which I have completely forgotten about them :-(. I change the PR to draft mode

**Pull request merged**